src/tcp/tcp.c

```diff
@@ -48,6 +48,10 @@
 #define SIZ_CAST
 #endif
 
+#ifdef WIN32
+#pragma comment(lib,  "ws2_32.lib") 
+#endif
+
```

Comment:
this gives a warning when using mingw32 compiler:

(ignoring #pragma comments)